src/index.html

```diff
@@ -104,6 +104,10 @@
         <title>soil moisture</title>
         <path id="soil_moisture" data-name="soil moisture" d="M6.62,13.19a2,2,0,0,1-1.68-.61c-.32-.32-.42-.68-.72-.9A2.44,2.44,0,0,1,2.68,14a3.47,3.47,0,0,0,.77-2.75,3.39,3.39,0,0,0-1.2-1.63A2.35,2.35,0,0,1,1.35,11a4.61,4.61,0,0,1,0,1,2.36,2.36,0,0,0,.15,1.59S.9,13.27,1,12.17a2.81,2.81,0,0,0-.06-.87A1.67,1.67,0,0,0,.1,12.71,1.53,1.53,0,0,1,.58,11a1.81,1.81,0,0,0,.74-2.23A3.47,3.47,0,0,1,.15,6.37C.15,3.62,2.56,2.19,2.56,0H4.33A6.57,6.57,0,0,1,3.15,3,1.72,1.72,0,0,1,4.61,4.14a3.73,3.73,0,0,1,1,.48c1,.64,1.55.5,1.55.5a1.47,1.47,0,0,1-1.59,0,3.37,3.37,0,0,0-.72-.26c.39,1.11.86,1.33.86,1.33a1.35,1.35,0,0,1-1.26-1C4,4,3.32,3.69,2.47,4a4,4,0,0,0-1,2.35c0,1.49,2.18,2.48,2.65,4.4.67.49.67.88,1.09,1.44C6.09,13.27,6.62,13.19,6.62,13.19ZM8.76,4.11c-.91,1.68-2.59,4.21-2.5,5.7a2.62,2.62,0,0,0,2.5,2.63,2.62,2.62,0,0,0,2.5-2.63C11.31,8.43,9.48,5.54,8.76,4.11ZM12.25,0c-.53,1-1.75,3.08-1.75,4A1.75,1.75,0,1,0,14,4C14,3,12.74,1,12.25,0Z"/>
       </symbol>
+      <symbol id="icon-emg-reading" viewBox="0 0 14 14">
+        <title>emg reading</title>
+        <path id="emg_reading" data-name="emg reading" d="M6.329,0.768L12.789,6.84L7.211,6.84L7.14,12.777L0.174,6.196L6.332,6.196L6.329,0.768Z"/>
+      </symbol>
```

Comment:
We're trying to move away from this practice of putting limited-use icons in the `index.html` where they're always present in the DOM even when they're not needed. Given that this seems to be a temporary icon, nothing needs to be done immediately but this should be viewed as a short-term approach.